packages/generator/src/generator.ts

```diff
@@ -299,13 +316,6 @@ export async function generateSourcesForService(
     );
   }
 
-  if (options.additionalFiles) {
```

Comment:
Perhaps I understand this condition wrong, but shouldn't we use your new function `generateAdditionalFiles` here? Or was it removed entirely intentionally?

Reply:
at this point we used `ts-morph` to add the file, which made reuse with openapi difficult. Since we want to replace `ts-morph` in the long run anyway, I decided to use basic `fs` also here, which lead to the removal here and inclusion in the non. ts-morph file.